hooks/useMap.ts

```diff
@@ -182,6 +187,7 @@ function initializeMap(map: mapboxgl.Map, data: any, pathogen: "Arbovirus" | "Sa
   );
 }
 
+//Hook called in MapAndFilters to initalize mapbox map, content, logic 
```

Comment:
I like the description, but the whole point of being reusable is we do not hardcode WHERE it is called so remove that (in the situation we call it elsewhere this comment becomes obsolete. that is the exact problem we want to avoid)

Reply:
Understood, removed the comment.